.github/workflows/smoke-tests.yml

```diff
@@ -3,11 +3,9 @@ name: Smoke tests
 on: [push]
 
 jobs:
-
```

Comment:
Sorry for these. Can we stick to this `yml` formatting convention, which is forced by Prettier?

Reply:
No worries, I don't mind sticking with some convention if it improves readability. In addition - is it possible to configure prettier for the project? I think it is worth adding it as a command to run in package.json and include it into "run all" command as well. That way it will be clear for all contributors why yaml files are structured one way or another. What do you think?

Reply:
Yes, it's definitely possible, and by the way Prettier will cover quite all the codebase here.

Reply:
But let's do this in separate PR, though